src/components/checkboxes/index.md.njk

```diff
@@ -65,6 +65,24 @@ You can add hints to checkbox items to provide additional information about the
 
 {{ example({group: "components", item: "checkboxes", example: "hint", html: true, nunjucks: true, open: false, size: "s"}) }}
 
+### Add an option for ‘none’
+
+In some cases, it can be helpful to require users to select a checkbox to say that none of the options apply to them, instead of leaving them all unchecked. This also makes sure users do not skip the question by accident.
+
+Show the “none” option last. Separate it from the other options using an “or” divider.
```

Comment:
```suggestion
Show the 'none' option last. Separate it from the other options using an `or` divider.
```
GOV.UK style guide advises only using double quotes for reported speech, and single quotes otherwise. Also, I suggested putting _or_ inside back quotes, as I think that's our convention when we mention code in guidance.

Reply:
I think backticks might be wrong here, as "or" isn't really a code reference. Maybe we should be consistent with how the Radios component describes this?

```suggestion
Show the ‘none’ option last. Separate it from the other options using a divider. The text is usually the word ‘or’.
```